cmd/root.go

```diff
@@ -63,6 +63,12 @@ func init() {
 	stringFlag(flags, "argocd-api-namespace", "ArgoCD namespace where the application watcher will read Custom Resource Definitions (CRD) for Application and ApplicationSet resources.",
 		newStringOpts().
 			withDefault("argocd"))
+	stringFlag(flags, "kubernetes-type", "Kubernetes Type One of eks, or localhost. Defaults to localhost.",
+		newStringOpts().
+			withChoices("eks", "localhost").
+			withDefault("localhost"))
```

Comment:
"localhost"? "local" might make more sense, since "localhost" normally refers to an ip address.

Reply:
changed to `local`